plugin-mongo/pom.xml

```diff
@@ -22,7 +22,7 @@
     <dependency>
         <groupId>org.mongodb</groupId>
         <artifactId>mongodb-driver-sync</artifactId>
-        <version>3.11.0</version>
+        <version>4.8.0-beta0</version>
```

Comment:
Is there a more stable version available?

Reply:
Both surprised and not surprised that the maven dependency updater thinks beta versions are acceptable. Thanks for pointing this out

Reply:
Fixed